### PR TITLE
feat: kusion init support custom params body in JSON string, replace default value in kusion.yaml of specified template

### PR DESCRIPTION
**Pull request by howieyuen**
close: https://github.com/KusionStack/kusion/issues/72

### CODE REVIEW

File: pkg/scaffold/project.go [hunk 1]
```diff
@@ -16,13 +19,103 @@ type ProjectTemplate struct {
...
+//      "array field":  []int{1, 2, 3},
+//      "map field": map[string]bool{
+//        "foo": true,
+//        "bar": false,
+//      },
+//      "inner struct": map[string]interface{}{
+//        "foo": "bar",
+//      },
+//    },
+//  }
```
Comment (chai2010) [inline pkg/scaffold/project.go:118]: the doc code is too large, put it into the example test code

**Review by elliotxx — APPROVED** [review]
lgtm

File: pkg/scaffold/project.go (hunk 1 above)
Comment (howieyuen) [reply pkg/scaffold/project.go]: ok, thanks for your advice

File: pkg/kusionctl/cmd/init/options.go [hunk 2]
```diff
@@ -126,60 +142,58 @@ func (o *InitOptions) Run() error {
...
-			// restore f type
-			actual, _ := f.RestoreActualValue(input)
-			configs[f.Name] = actual
+			tc.ProjectConfig[f.Name] = actual
+		}
+
+		// Prompt stack configs from kusion.yaml
+		tc.StacksConfig = make(map[string]map[string]interface{})
+		for i := range template.StackTemplates {
+			stack := template.StackTemplates[i]
```
Comment (chai2010) [inline pkg/kusionctl/cmd/init/options.go:165]: `for _, stack := ...`

**Review by chai2010 — APPROVED** [review]
LGTM

**Review by elliotxx — APPROVED** [review]
lgtm

**Pull request merged**